src/main/java/spoon/support/reflect/declaration/CtElementImpl.java

```diff
@@ -369,9 +369,13 @@ public void enter(CtElement e) {
 
 	@Override
 	public String toString() {
-		DefaultJavaPrettyPrinter printer = new DefaultJavaPrettyPrinter(getFactory().getEnvironment());
-		printer.computeImports(this);
-		printer.scan(this);
+		DefaultJavaPrettyPrinter printer = new DefaultJavaPrettyPrinter(
+				getFactory().getEnvironment());
+		try {
+			printer.computeImports(this);
+			printer.scan(this);
+		} catch (ParentNotInitializedException ignore) {
+		}
```

Comment:
add something in the result string: <parent error to print the rest if the element>